app/controllers/hub/clients_controller.rb

```diff
@@ -31,11 +31,11 @@ def create
     def show; end
 
     def edit
-      @form = ClientIntakeForm.from_intake(@client.intake)
+      @form = UpdateClientForm.from_client(@client.intake)
```

Comment:
The name of the forms (ClientIntakeForm & CreateClientForm) caused a lot of confusion to Alex and I when working on it, so updating to be a lot more clear on what the purpose of each form was made some sense to us. Also, since they are now inheriting from the same "ClientForm" class, it also made more sense from a naming and consistency standpoint pass the client to the form instead of the intake.

Reply:
I love this rename!